packages/design-system/src/components/web-components/ds-tooltip/ds-tooltip.tsx

```diff
@@ -0,0 +1,106 @@
+import { define } from '../preactement/define';
+import { Tooltip, TooltipProps } from '../../Tooltip';
+import { Placement } from '@popperjs/core';
+import { parseBooleanAttr } from '../wrapperUtils';
+
+const attributes = [
+  'active-class-name',
+  'class-name',
+  'close-button-label',
+  'component',
+  'content-heading',
+  'dialog',
+  'interactive-border',
+  'inversed',
+  'max-width',
+  'placement',
+  'root-id',
+  'show-close-button',
+  'title',
+  'transition-duration',
+  'trigger-aria-label',
+  'z-index',
+] as const;
+
+/* eslint-disable @typescript-eslint/no-namespace */
+declare global {
+  namespace JSX {
+    interface IntrinsicElements {
+      'ds-tooltip': React.DetailedHTMLProps<
+        React.HTMLAttributes<HTMLElement> & {
+          [K in (typeof attributes)[number]]?: string;
+        },
+        HTMLElement
+      >;
+    }
+  }
+}
+/* eslint-enable */
+
+interface WrapperProps
+  extends Omit<
+    TooltipProps,
+    | 'ariaLabel'
+    | 'contentHeading'
+    | 'dialog'
+    | 'inversed'
+    | 'showCloseButton'
+    | 'offset'
+    | 'placement'
+    | 'title'
+  > {
+  contentHeading?: string | TooltipProps['contentHeading'];
```

Comment:
Do you have a story to demonstrate that it's possible to pass in a `ReactNode` (e.g. what `TooltipProps['contentHeading']` implies here)?

Reply:
So interestingly the React.ReactNode type has the following definition:

```
type ReactNode =
        | ReactElement
        | string
        | number
        | Iterable<ReactNode>
        | ReactPortal
        | boolean
        | null
        | undefined
        | DO_NOT_USE_OR_YOU_WILL_BE_FIRED_EXPERIMENTAL_REACT_NODES[
            keyof DO_NOT_USE_OR_YOU_WILL_BE_FIRED_EXPERIMENTAL_REACT_NODES
        ];
```
So actually adding this as an available type is too permissive on the web component. I think the compiler was yelling at me, let me see if I can figure another way to satisfy Typescript.

Reply:
> DO_NOT_USE_OR_YOU_WILL_BE_FIRED_EXPERIMENTAL_REACT_NODES

😂